formfiller/src/main/java/com/diyasylum/formfiller/application/models/AddressWithDates.java

```diff
@@ -0,0 +1,108 @@
+package com.diyasylum.formfiller.application.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+
+public class AddressWithDates {
+  private final String streetName;
+  private final String streetNumber;
+  private final String cityOrTown;
+  private final String departmentProvinceOrState;
+  private final String country;
+  private final String fromDate; // mm/yyyy
```

Comment:
Part of me wants to make this a separate class `MmYyyyDate` to enforce good formatting but whatever we can just run a validator

Reply:
How would you implement the class? Should I accept a String or a Java date type as the input and have a getter which gives the correctly formatted string?

Reply:
I was thinking a month string and a year string. Or just ints. But why bother? We can make sure it's right on the FE